modules/dxp/apps/saml/saml-api/src/main/java/com/liferay/saml/constants/SamlWebKeys.java

```diff
@@ -77,6 +79,9 @@
 
 	public static final String SAML_SSO_SESSION_ID = "SAML_SSO_SESSION_ID";
 
+	public static final String SAML_SUBJECT_SCREEN_NAME =
+		"SAML_SUBJECT_SCREEN_NAME";
```

Comment:
@csierra  I just remembered that I actually intended to rename this attribute to SAML_SUBJECT_NAME_ID to be more aligned with the SAML concept. Just for consideration.